cargo-near/src/util/tracking.rs

```diff
@@ -0,0 +1,58 @@
+use reqwest::Client;
+use std::{env, str};
+use tracing::debug;
+
+const SEND_TRACKING_REQUEST_ERROR: &str = "Can't send tracking usage event";
+
+#[derive(Debug, serde::Serialize)]
+struct MixpanelProperties {
+    token: String,
+    pkg_version: String,
+    os: String,
+}
+
+#[derive(Debug, serde::Serialize)]
+struct TrackingData {
+    event: String,
+    properties: MixpanelProperties,
+}
+
+pub(crate) fn track_usage() {
+    let properties = MixpanelProperties {
+        token: "24177ef1ec09ffea5cb6f68909c66a61".to_string(),
+        pkg_version: env!("CARGO_PKG_VERSION").to_string(),
+        os: env::consts::OS.to_string(),
+    };
+    let tracking_data = TrackingData {
+        event: "CNN".to_string(),
+        properties,
+    };
+
+    let client = Client::new();
+
+    if tokio::runtime::Runtime::new()
+        .unwrap()
+        .block_on(
+            client
+                .post("https://api.mixpanel.com/track")
```

Comment:
the test passes for arbitrary path `https://api.mixpanel.com/tracker/anything/else/arbitrary/path`  too

Reply:
Yep, also looks like the mixpanel endpoint returns 200 both with the right token and with random one

Reply:
But I tested it and the test itself works. If request fails the test fails as well